metaflow/plugins/kfp/dockerfiles/integration_testing_full_image/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM hsezhiyan/metaflow-zillow:1.6
```

Comment:
I suggest I do this in a separate PR: Dockerhub is not possible because of rate limits, Github Packages/Container Registry because of an auth issue which cannot be bypassed even if the image is public. AWS ECR may be possible, but I'll be investigate. Right now, we have a working Artifactory solution which I can place here.